### PR TITLE
Add What's New for ECSV enhancement

### PR DESCRIPTION
**Pull request by taldcroft**
Also fixes a minor omission in ascii.write docs for ECSV.

### CODE REVIEW

File: docs/io/ascii/write.rst [hunk 1]
```diff
@@ -298,6 +298,7 @@ Starting with astropy 2.0 it is possible to store not only standard
 - `astropy.coordinates.Latitude`
 - `astropy.coordinates.Longitude`
 - `astropy.coordinates.Angle`
+- `astropy.coordinates.distances.Distance`
```
Comment (mhvk) [inline docs/io/ascii/write.rst]: Why not just `astropy.coordinates.Distance`?

File: docs/whatsnew/2.0.rst [hunk 2]
```diff
@@ -300,6 +301,27 @@ read from/write to FITS files, provides methods for arithmetic operations
...
+<ecsv_format>` table file.  The table file can then be read back
+into astropy with no loss of object data or attributes.
+
+- `astropy.time.Time`
+- `astropy.time.TimeDelta`
+- `astropy.units.Quantity`
+- `astropy.coordinates.Latitude`
+- `astropy.coordinates.Longitude`
+- `astropy.coordinates.Angle`
+- `astropy.coordinates.distances.Distance`
```
Comment (mhvk) [inline docs/whatsnew/2.0.rst]: Same here.

File: docs/io/ascii/write.rst (hunk 1 above)
Comment (taldcroft) [inline docs/io/ascii/write.rst]: Oops, I just copied from `serialize.py`.  But I agree with the change.

**Comment by mhvk** [comment]
Great, merging!

**Pull request merged**